mws/acceptance/log_delivery_test.go

```diff
@@ -0,0 +1,60 @@
+package acceptance
+
+import (
+	"os"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/internal/acceptance"
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestMwsAccLogDelivery(t *testing.T) {
+	if os.Getenv("CLOUD_ENV") != "MWS" {
+		t.Skip("Cannot run test on non-MWS environment")
+	}
+	acceptance.AccTest(t, resource.TestCase{
+		Steps: []resource.TestStep{
+			{
+				Config: qa.EnvironmentTemplate(t, `
+				provider "databricks" {
+					host     = "{env.DATABRICKS_HOST}"
+					username = "{env.DATABRICKS_USERNAME}"
+					password = "{env.DATABRICKS_PASSWORD}"
+				}
+			
+				resource "databricks_mws_credentials" "ld" {
+					account_id       = "{env.DATABRICKS_ACCOUNT_ID}"
+					credentials_name = "tf-acceptance-logdelivery-{var.RANDOM}"
```

Comment:
nit: names in this and other resources can be duplicated across resources so should be fine even if we dont introduce randomness here.

Reply:
@svar29 RANDOM identifier here is more to tie different resources created by same test in case of debugging.